### ISSUE TITLE
Review TCP branch 3.5.2-dev

### ISSUE THREAD

**Issue by chancecorbeil** [Trillium-Solutions/transit-custom-posts#110]
Hi @ekeaton Can you please review all the new updates on TCP?

All these updates are read to review on a single branch ( 3.5.2-dev ). I was developing these branches as separate updates as we normally do, but many of the updates required the last update to function, so I decided to start merging all the updates into a branch so I could continue forward. All your reviews can be done on 3.5.2-dev.

### Updates
- [x] #95
- [x] #94
- [x] #93
- [x] #101
- [x] #92
- [x] #83
- [x] #106

**Comment by ekeaton** [comment]
@chancecorbeil Notes:

When I activated the plugin I received this error:

> The plugin generated 47 characters of unexpected output during activation. If you notice “headers already sent” messages, problems with syndication feeds or other issues, try deactivating or removing this plugin.

I was able to ignore the message and still use the plugin but here's a link about what that means https://stackoverflow.com/questions/4074477/the-plugin-generated-x-characters-of-unexpected-output-during-activation-wordpr. 

- The alerts panel isn't opening when using the collapse and use button options from the TCP settings menu.
- The 'enable' button feature that works with 'enable collapse' (for alerts) appears after you select collapse and click save. This could be easy to miss if someone left the page right after. Maybe a note below the "enable collapse" option would help, telling the user to go back after clicking save to also enable the button option? It looks like you mention this in issue #101. A note could be useful for that area as well. 
- I selected to add a link in the alerts option and the collapse panel opened, however, the text for the alert was missing and only a link saying "read more" showed. We will still want the alert text to appear on route alerts when the panel is opened.

**Comment by ekeaton** [comment]
@chancecorbeil 

> I selected to add a link in the alerts option and the collapse panel opened, however, the text for the alert was missing and only a link saying "read more" showed. We will still want the alert text to appear on route alerts when the panel is opened.

I believe the missing text bug that occurred on the two sites I tested on should be fine now and was related to updated field names. Single alert desc --> route alert desc etc.

**Comment by chancecorbeil** [comment]
@ekeaton Oh great! I tested that too, and all contents transitioning from 3.5.1 to 3.5.2 are looking fine to me.

For Ellensburg, the GTFS import bug where only 1 route appears is happening on all versions of TCP. I've tried on a local copy of Production using version 2, the dev environment on 3.4, 3.5.1, and 3.5.2. On all GTFS imports only that single route appears. I can see 5 other routes in the GTFS though.
